build/LayoutNugetPackage.csproj

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Layout">
```

Comment:
The NuGet layout can be produced locally just by building this project i.e. _msbuild build\LayoutNuGetPackage.csproj_
I'll change the project file extension to _.proj_ as it isn't a C# project.